modules/sdk-coin-bsc/src/lib/resources.ts

```diff
@@ -0,0 +1,22 @@
+import EthereumCommon from '@ethereumjs/common';
+import { coins, EthereumNetwork } from '@bitgo/statics';
+
+export const testnetCommon = EthereumCommon.forCustomChain(
```

Comment:
Should these be in `statics`

Reply:
Hmm, all the other eth-like coins have this `resources.ts` file in their respective modules. Should I still move it over to statisc?

Reply:
Can you give me an example you are replicating?

Reply:
Testnet definitions would make more sense here: `modules/statics/src/networks.ts`

Reply:
An example (for Celo) is: https://github.com/BitGo/BitGoJS/blob/master/modules/sdk-coin-celo/src/lib/resources.ts
It seems that all the eth-like coins (Etc, Celo, Rbtc, and Avaxc) all have this common object defined in `resources.ts`.

So I already have the mainnet and testnet definitions in `modules/statics/src/networks.ts`, but what I could do is add a few properties to them, and then create the common object dynamically in this function: https://github.com/BitGo/BitGoJS/blob/1c27a6343f30d341588eadd2a323b4ac2fe73646/modules/sdk-coin-bsc/src/lib/utils.ts#L15

Reply:
On second thought, I think it is best to keep it as is. It would require inconsistent changes in statics.

Reply:
Thanks for the references :)
I appreciate it